doc/8-advanced/commandline.md

```diff
@@ -0,0 +1,38 @@
+# command line
+
+to start furnace at the command line, use the following format:
+
+`furnace [params] [filename]`
+
+the following parameters may be used:
+
+- `-help`: display help.
+- `-audio <engine>`: set audio engine to one of the following:
+  - `jack`: JACK. only on Linux.
+  - `sdl`: SDL. default.
+  - `portaudio`: PortAudio.
+- `-output <filename>`: output audio to `filename`.
+- `-vgmout <filename>`: output .vgm data to `filename`.
+- `-direct`: set VGM export direct stream mode.
+- `-zsmout <filename>`: output .zsm data for Commander X16 Zsound.
+- `-cmdout <filename>`: output command stream.
+- `-binary`: set command stream output format to binary.
+- `-loglevel <level>`: set the logging level to one of the following:
+  - `error`: critical errors only.
+  - `warning`: errors and warnings.
+  - `info`: errors, warnings, and useful information. default.
+  - `debug`: most verbose, with all of the above and more.
```

Comment:
There also is `trace`, which is even more verbose than debug.

Reply:
How about this:
- `debug`: all of the above and information useful for debugging.
- `trace`: most verbose. includes details of inner workings. default.